.github/workflows/scripts/validate-coderabbit.sh

```
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+echo "Step 1: Checking CodeRabbit.ai approval..."
+
+echo "Fetching PR reviews from GitHub API..."
+reviews=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+  "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews")
```

Shellcheck (0.10.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)